task_manager/tests/test_user.py

```diff
@@ -0,0 +1,221 @@
+import pytest
+from django.test import Client
+from freezegun import freeze_time
+from task_manager.tests.paths import resolve_path
+from task_manager.users.models import ServiceUser
+
+pytestmark = pytest.mark.django_db
+
+
+def test_users_list(client: Client, service_user: ServiceUser) -> None:
+    # given
+    route = "/users/"
+
+    # when
+    response = client.get(route)
+
+    # then
+    expected_html = resolve_path("fixtures/users.html")
+    assert response.content.decode() == expected_html.read_text()
+    assert response.status_code == 200
+
+
+def test_create_user_get(client: Client) -> None:
+    # given
+    route = "/users/create/"
+
+    # when
+    response = client.get(route)
+
+    # then
+    # csrfmiddlewaretoken is different each time
+    assert (
+        "Для подтверждения введите, пожалуйста, пароль ещё раз."
+        in response.content.decode()
+    )
+    assert response.status_code == 200
```

Comment:
I guess we're missing a test case when user is logged in

Reply:
it's probably "authenticated_user / unauthenticated_user"  that makes some a mess.
All casses are covered by tests.
It'll be better to